cli-toolchain/quickstart.md

```diff
@@ -931,8 +926,8 @@ Displays a list of all the environments in your Amplify project
 Removes an environment from the Amplify project
 * amplify env get --name <env-name> <br>
 Displays the details of the environment specified in the command 
-* amplify env pull --restore <br>
-Pulls your environment with the current cloud environment. Use the restore flag to overwrite your local backend configs with that in the cloud.
+* amplify env pull <br>
+Pulls your environment with the current cloud environment without impacting your local changes. Add the `--restore` flag to overwrite your local backend config with that in the cloud (similar to the `amplify pull` command).
```

Comment:
"Pulls your environment with the current cloud environment without impacting your local changes." --> This sentence is confusing to me. 
Do you mean to say "Pulls you environment from the cloud ...."

Reply:
Just read this sentence and it makes no sense. Thanks for catching it. Updating.